css/style.css

```diff
@@ -349,87 +376,119 @@ header {
+    transition: color, 0.5s; }
+    
+  .cipher-service h2 {
+    font-size: 30px;
+    font-size: 1.875rem; }
+    
+  .cipher-service p {
+    font-size: 18px;
+    font-size: 1.125rem;
+    line-height: 1.6; }
+
+  .cipher-service.focus h2 {
+    color: #ffffff; }
+    
+  .cipher-service.focus p {
+    color: rgba(255, 255, 255, 0.5); } }
+
+  .separator {
+    position: relative;
+    height: 300px;
+    background-image: url(../img/pineforestdusk.png);
+    z-index: 2;
+    background-size: cover;
+    background-repeat: no-repeat; }
 
 @media screen and (max-width: 991px) {
   .cipher-background-wrapper .foreground .hero_content {
     height: 400px; }
-    .cipher-background-wrapper .foreground .hero_content .hero_title h4 {
+    
+  .cipher-background-wrapper .foreground .hero_content .hero_title h4 {
       padding-bottom: 40px; }
-    .cipher-background-wrapper .foreground .hero_content .hero_title h1 {
+    
+  .cipher-background-wrapper .foreground .hero_content .hero_title h1 {
       font-size: 80px; }
-    .cipher-background-wrapper .foreground .hero_content .hero_title img {
+    
+  .cipher-background-wrapper .foreground .hero_content .hero_title img {
       width: 100px; } }
+
 @media screen and (max-width: 680px) {
   .cipher-background-wrapper .foreground .hero_content {
     height: 300px; }
-    .cipher-background-wrapper .foreground .hero_content .hero_title h4 {
+
+  .cipher-background-wrapper .foreground .hero_content .hero_title h4 {
       padding-bottom: 30px; }
-    .cipher-background-wrapper .foreground .hero_content .hero_title h1 {
+    
+  .cipher-background-wrapper .foreground .hero_content .hero_title h1 {
       font-size: 60px; }
-    .cipher-background-wrapper .foreground .hero_content .hero_title img {
+    
+  .cipher-background-wrapper .foreground .hero_content .hero_title img {
       width: 80px; } }
+
 @media screen and (max-width: 465px) {
   .cipher-background-wrapper .foreground .hero_content {
     height: 50px;
     padding-top: 10%; }
-    .cipher-background-wrapper .foreground .hero_content .hero_title {
+    
+  .cipher-background-wrapper .foreground .hero_content .hero_title {
       padding-bottom: 20px; }
-      .cipher-background-wrapper .foreground .hero_content .hero_title h4 {
+    
+  .cipher-background-wrapper .foreground .hero_content .hero_title h4 {
         padding-bottom: 5px; }
-      .cipher-background-wrapper .foreground .hero_content .hero_title h1 {
+    
+  .cipher-background-wrapper .foreground .hero_content .hero_title h1 {
         font-size: 40px; }
-      .cipher-background-wrapper .foreground .hero_content .hero_title img {
+    
+  .cipher-background-wrapper .foreground .hero_content .hero_title img {
         width: 60px; }
-    .cipher-background-wrapper .foreground .hero_content .use-btn {
+    
+  .cipher-background-wrapper .foreground .hero_content .use-btn {
       display: none; } }
 
 /*# sourceMappingURL=style.css.map */
+.sub {
```

Comment:
not sure why this didn't transfer to my commit -- did you delete this after committing with SASS?